Tests/ResponseTests.swift

```diff
@@ -672,8 +672,27 @@ final class ResponseTryMapErrorTestCase: BaseTestCase {
         else { XCTFail(); return }
 
         XCTAssertEqual(underlyingError.asAFError?.isSessionTaskError, true)
-        XCTAssertEqual((underlyingError.asAFError?.underlyingError as? URLError)?.code, .cannotFindHost)
-
+        XCTAssertEqual(underlyingError.asAFError?.isHostURLError, true)
         XCTAssertNotNil(response?.metrics)
     }
 }
+
+private extension DataResponse where Failure == AFError {
```

Comment:
We prefer extensions on the `AFError` type itself, not the response type, otherwise we need separate extensions for every response.